docs/user/openstack/add_worker_nodes_by_hand.md

```diff
@@ -0,0 +1,69 @@
+# Adding Compute Nodes By Hand
+The preferred method of adding compute nodes to an OpenShift cluster is
+to use the worker MachineSet with [Machine API Operator](https://github.com/openshift/machine-api-operator).
+
+It is also possible to add a compute node through other tools like ansible. For example, 
+the compute nodes may be added as described in [Installing OpenShift on OpenStack User-Provisioned Infrastructure][1];
+or when scaling up the compute nodes using [Openshift-ansible][2].
+
+When doing so, there are a few important points that should be taken into account. 
+
+## Apply Necessary Security Groups To Ports
+When adding a new compute, the necessary security groups need to be applied to 
+allow the node access the cluster API endpoints and services.   
+
+If the node is directly attached to the machineNetwork,  the pre-existing worker security groups maybe 
+applied to the port attaching the node. The existing security rules should not require modifications.  
+
+If the nodes is attached to a different network than the machineNetwork,
+then new security group rules will have to be configured because the security group rules deployed by default on a cluster
+match on both destination and source networks.
+
+
+## Configure Allowed Address Pairs On Node Port
+The correct address pairs need to be configured on the port attaching the new node to the machineNetwork.  
+The allowed address pairs should configure the port to accept traffic for
+ingress VIP.  
+For example: 
+
+
+```
+
+$ openstack port set 97f73ba7-e104-49e2-ad7a-d4b440acc57e --allowed-address ip-address='10.0.128.7'
+$
+$ openstack port show 97f73ba7-e104-49e2-ad7a-d4b440acc57e -c allowed_address_pairs
+   +-----------------------+----------------------------------------------------------+
+   | Field                 | Value                                                    |
+   +-----------------------+----------------------------------------------------------+
+   | allowed_address_pairs | ip_address='10.0.128.7', mac_address='fa:16:3e:0d:35:f8' |
+   +-----------------------+----------------------------------------------------------+
+```
+
+*Note: When configuring the allowed address pairs the MAC address should *not* be specified. 
+For example: 
```

Comment:
Not needed, because we already provide the command to set the allowed address pair on the port.